libqtile/state.py

```diff
@@ -45,11 +45,11 @@ def apply(self, qtile):
         Rearrange the windows in the specified Qtile object according to this
         QtileState.
         """
-        for (group, layout) in self.groups.items():
+        for (group, layout) in self.groups:
```

Comment:
Wouldn't this break when upgrading from older versions? Not sure what's the best way to handle that though.